api-docs/components/schemas/event.json

```diff
@@ -259,27 +270,20 @@
           }
         }
       },
-      "contexts": {
-        "type": "object"
-      },
+      "contexts": { "$ref": "#/contexts" },
       "fingerprints": {
         "type": "array",
         "items": {
           "type": "string"
         }
       },
-      "context": {
-        "type": "object"
-      },
+      "context": { "$ref": "#/context" },
       "release": {
         "type": "object",
         "nullable": true,
-        "oneOf": [
+        "allOf": [
           {
             "$ref": "releases/organization-release.json#/OrganizationRelease"
```

Comment:
It's possible to specify a null value without needing a separate null reference, see https://stackoverflow.com/questions/40920441/how-to-specify-a-property-can-be-null-or-a-reference-with-swagger